inc/template.php

```diff
@@ -1864,6 +1866,25 @@ function tpl_classes() {
         (isset($INFO) && $INFO['exists']) ? '' : 'notFound',
         ($ID == $conf['start']) ? 'home' : '',
     );
+    $id=$ID;
+    if($id[0] == ':')
+        $id = substr($id,1);
```

Comment:
Please wrap the statement of the if with {}

Reply:
yes sorry for that. I'm pushing a little commit to correct that.